wolfcrypt/src/wc_port.c

```diff
@@ -3668,11 +3668,13 @@ char* mystrnstr(const char* s1, const char* s2, unsigned int n)
 
 #elif defined(WOLFSSL_ZEPHYR)
 
+    void* wolfsslThreadHeapHint = NULL;
```

Comment:
Consider using the mechanic being introduced in https://github.com/wolfSSL/wolfssl/pull/7478. This would allow your code to continue to pass NULL's in here.

Reply:
I'll wait for that PR to merge and use it instead. Good idea.